### PR TITLE
Remove InstallEvent.activeWorker

### PR DESCRIPTION
**Pull request by Elchi3**
For https://github.com/mdn/browser-compat-data/pull/20138

### CODE REVIEW

File: files/en-us/web/api/installevent/activeworker/index.md
Comment (bsmth) [inline files/en-us/web/api/installevent/activeworker/index.md:1]: Should we add a redirect to `files/en-us/web/api/installevent/index.md` or is that unnecessary?

**Review by bsmth — APPROVED** [review]
Looks good to me with one comment for consideration before merging 👍🏻 

Thank you!

File: files/en-us/web/api/installevent/activeworker/index.md
Comment (Elchi3) [reply files/en-us/web/api/installevent/activeworker/index.md:1]: Added 👍

**Comment by Elchi3** [comment]
Can this get merged @bsmth?

**Comment by bsmth** [comment]
Thanks @Elchi3 👍

**Pull request merged**